core/src/main/java/org/ehcache/core/EhcacheManager.java

```diff
@@ -490,9 +490,9 @@ public void close() throws Exception {
         .getServiceConfigurations()
         .toArray());
     if (eventSourceConfiguration != null) {
-      eventParallelism = eventSourceConfiguration.getOrderedEventParallelism();
+      eventParallelism = eventSourceConfiguration.getDispatcherConcurrency();
```

Comment:
There are 4 occurrences of the variable name eventParallelism that should be changed.